msgpack.d.ts

```diff
@@ -1,16 +1,28 @@
 /**
  * Serializes a value to a MessagePack byte array.
- * @param data The value to serialize. This can be a scalar, array or object.
- * @param multiple Indicates whether multiple values in data are concatenated to multiple MessagePack arrays.
+ * @param data An object that defined additional options.
+ * @param options Indicates whether multiple values in data are concatenated to multiple MessagePack arrays.

```

Comment:
I guess that went in the wrong line?
```suggestion
 * @param data The value to serialize. This can be a scalar, array or object.
 * @param options An object that defines additional options.
```

Reply:
Good catch. Fixed.